docs/howto/custom-template-tags.txt

```diff
@@ -788,6 +788,19 @@ Notes:
   ``django.template.TemplateSyntaxError`` if they receive the wrong number or
   type of arguments.
 
+.. versionadded:: 1.9
```

Comment:
Try to make the note "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.